src/main/java/tech/gruppone/stalker/server/model/api/UserDataDto.java

```diff
@@ -17,5 +21,5 @@
 
   @NonNull LocalDate birthDate;
 
-  @NonNull Timestamp creationDateTime;
+  Timestamp creationDateTime;
```

Comment:
```suggestion
  @With
  Timestamp creationDateTime;
```

se proprio proprio non ti funziona aggiungi qui il with:

https://projectlombok.org/features/With

che è essenzialmente un setter per valori immutabili